accounts/templates/accounts/profile_settings.html

```diff
@@ -2,7 +2,6 @@
 
 <link rel="stylesheet" href="{% static 'accounts/css/settings.css' %}" />
 <link rel="stylesheet" href="{% static 'accounts/css/modal.css' %}" />
-<script src="{% static 'accounts/js/modal.js' %}" defer></script>
```

Comment:
don't we still need the modal for stuff like the confirmation on deleting the account?

Reply:
Yeah the modal style class was kept but I removed/repurposed the javascript file because it could only handle if there was 1 modal box on a page (due to class wide styling/actions instead of specific buttons -> specific modals)